app/views/presentations/_errors.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if @presentation.errors.any? %>
+  <div class="error-explanation">
+    <h2 class="error-explanation__header">
```

Comment:
Nope, will remove. Legacy stuff from when I was starting the project. Since I've been recycling the _error partial I created then.